clients/wrangler/clients/clients.go

```diff
@@ -0,0 +1,99 @@
+package clients
+
+import (
+	"context"
+	"time"
+
+	"github.com/rancher/shepherd/pkg/session"
+	"github.com/rancher/shepherd/pkg/wrangler"
+	"github.com/rancher/wrangler/v2/pkg/ratelimit"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/dynamic"
+
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+type Clients struct {
+	*wrangler.Context
+	Dynamic dynamic.Interface
```

Comment:
We don't need this here. We have our own

Reply:
This is just for setting up the connection between Rancher and the Shepherd. This client is not performing a test cleanup. It is all handled by the controllers

Reply:
This is used a completely separate client, I promise you it doesn't need to be here

Reply:
It's true!